src/metrics/cilisi/script.R

```diff
@@ -0,0 +1,48 @@
+library(anndata)
+library(scIntegrationMetrics)
+
+## VIASH START
+par <- list(
+  input_integrated = "resources_test/task_batch_integration/cxg_immune_cell_atlas/integrated_processed.h5ad",
+  input_solution = "resources_test/task_batch_integration/cxg_immune_cell_atlas/solution.h5ad",
+  output = "output.h5ad"
+)
+meta <- list(
+  name = "cilisi"
+)
+## VIASH END
+
+cat("Reading input files\n")
+adata <- anndata::read_h5ad(par[["input_integrated"]])
+solution <- anndata::read_h5ad(par[["input_solution"]])
+embeddings <- adata$obsm[["X_emb"]]
+metadata <- solution$obs
+
+cat("Compute CiLISI metrics...\n")
+lisisplit <-
+  scIntegrationMetrics::compute_lisi_splitBy(
+                                            X = embeddings,
+                                            meta_data = metadata,
+                                            label_colnames = "batch",
+                                            perplexity = 30,
+                                            split_by_colname = "cell_type",
+                                            normalize = TRUE,
+                                            min.cells.split = 10,
+                                            min.vars.label = 2)
```

Comment:
Nitpick: reducing indents would be nice

```suggestion
  scIntegrationMetrics::compute_lisi_splitBy(
    X = embeddings,
    meta_data = metadata,
    label_colnames = "batch",
    perplexity = 30,
    split_by_colname = "cell_type",
    normalize = TRUE,
    min.cells.split = 10,
    min.vars.label = 2
)
```

Reply:
You can run [**{styler}**](https://styler.r-lib.org/) or [**air**](https://posit-dev.github.io/air/) to automatically format things if that's helpful

Reply:
Are you envisioning e.g. a pre-commit hook that does this for all files?